master/buildbot/steps/vstudio.py

```diff
@@ -415,7 +415,9 @@ def start(self):
         if self.platform is None:
             config.error('platform is mandatory. Please specify a string such as "Win32"')
 
-        command = ["%VCENV_BAT%",
```

Comment:
please put that in another PR, so that we can merge it faster. Looks like this one needs more tests, according to @sa2ajj